app/services/foods_service.rb

```diff
@@ -0,0 +1,18 @@
+class FoodsService
+    def self.get_url(url, keyword = nil)
+        conn = Faraday.new(url: "https://api.nal.usda.gov") do |f|
+            f.headers[:'x-api-key'] = ENV['api_key']
+            f.params[:query] = keyword unless keyword.nil?
```

Comment:
you might also add the param for [:totalHits] here as well. So that you don't have to make two separate calls.